core/indigo-core/molecule/multiple_cdx_loader.h

```diff
@@ -23,6 +23,10 @@
 #include "base_cpp/reusable_obj_array.h"
 #include "base_cpp/tlscont.h"
 
+typedef unsigned short int UINT16;
```

Comment:
do we need it in two places? (CDX constants)?

Reply:
deleted. (This definition was moved here because of a build failure and UINT16 was in error messages as undefined.)